snap/info.go

```diff
@@ -772,6 +776,12 @@ type MediaInfo struct {
 
 type MediaInfos []MediaInfo
 
+type UsernameInfo struct {
```

Comment:
this is badly placed, it's disrupting MediaInfos definitions. Also I think it should have the full name "SystemUsername", we have other kind of usernames (account ones), not sure the suffix Info is needed though.